packages/p2p/src/pages/advertiser-page/advertiser-page-row.jsx

```diff
@@ -70,6 +70,14 @@ const AdvertiserPageRow = ({ row: advert }) => {
         }
     };
 
+    React.useEffect(() => {
+        const disposeAdvertIntervalReaction = buy_sell_store.registerAdvertIntervalReaction();
+
+        return () => {
+            disposeAdvertIntervalReaction();
+        };
+    }, []);
+
```

Comment:
this is done so we can subscribe to p2p_advert_info when viewing an advertisers adverts from advertiser page. previously it didn't subscribe to the endpoint. This is similar to the one in Buy/Sell page, the function handles the subscription